goss-testing/scripts/check_clock_skew.sh

```diff
@@ -44,42 +44,42 @@ for node in $nodes; do
   pdsh_node_args="$pdsh_node_args -w $node"
 done
 
-for try in {1..3}; do
-  # Set/reset variables at the top of each attempt loop
-  cnt=0
-  exit_code=0
-  baseline=""
+# Set/reset variables at the top of each attempt loop
```

Comment:
Shouldn't this comment be changed, since this PR also removes the retry loop?
Also, was it intentional to remove the retry loop? The PR description only talks about closing the persistent connection. Is there a reason we want the retries removed?

Reply:
Ok, I've run it in a loop on a few systems for a while now and not seen any failures, so I do agree that the loop appears unneeded. I'll update my review to indicate that just the comment should be changed.